openedx/features/caliper_tracking/transformers/exam_transformers.py

```diff
@@ -96,3 +96,34 @@ def edx_special_exam_timed_attempt_started(current_event, caliper_event):
     })
     caliper_event['referrer']['type'] = 'WebPage'
     return caliper_event
+
+
+def edx_special_exam_practice_attempt_created(current_event, caliper_event):
+    """
+    The server emits this event when a learner chooses to take a special exam.
+
+    :param current_event: default event log generated.
+    :param caliper_event: caliper_event log having some basic attributes.
+    :return: updated caliper_event.
+    """
+    assessment_name = current_event['event'].pop('exam_name')
+    caliper_event.update({
+        'type': 'NavigationEvent',
+        'action': 'NavigatedTo',
+        'object': {
+            'id': current_event['referer'],
+            'type': 'WebPage',
+            'name': assessment_name,
```

Comment:
You don't need a separate variable for this.
```suggestion
             'name': current_event['event'].pop('exam_name'),
             
```

Reply:
done